src/icon-manifest/manifestMerger.ts

```diff
@@ -87,7 +87,7 @@ export function toggleAngularPreset(
   disable: boolean,
   files: models.IFileCollection): models.IFileCollection {
   const icons = files.supported
-    .filter(x => x.icon.startsWith('ng_') && (x.icon.endsWith('_js') || x.icon.endsWith('_ts')))
+    .filter(x => x.icon.startsWith('ng_') && !x.icon.endsWith('2'))
```

Comment:
2. Maintainability. You don't need to adjust the core code, but it works on the fly which is another principle of S.O.L.I.D. code.